mathics/builtin/assignment.py

```diff
@@ -1031,23 +1031,37 @@ class Quit(Builtin):
     >> a = 3
      = 3
     >> Quit[]
+     = Null[]
+    >> a
+     = a
+
+
+    >> a = 3
+     = 3
+    >> Quit
     >> a
      = a
 
     'Quit' even removes the definitions of protected and locked symbols:
     >> x = 5;
     >> Attributes[x] = {Locked, Protected};
     >> Quit[]
+     = Null[]
     >> x
      = x
     """
-
     def apply(self, evaluation):
         'Quit[]'
 
         evaluation.definitions.set_user_definitions({})
         return Symbol('Null')
 
+    def apply2(self, evaluation):
+        'Quit'
+
+        evaluation.definitions.set_user_definitions({})
+        return Symbol('Null')
+
```

Comment:
Looks like there are two things being changed in this PR. Can you split this up into just the natlang changes please?